R/order_views.R

```diff
@@ -1,27 +0,0 @@
-set_view_order <- function(views, order){
```

Comment:
no we don't, other than separating sides and views vs the globals. We might want to re-think how ordering should be applied and enforced. Right now it is first come first plotted except for axis and legend (first and last).